### PR TITLE
feat: provide granular progress of starlark package run

### PR DESCRIPTION
**Pull request by adschwartz**
## Description:
* Refactor code to move out of Log.js into the CreateEnclaveLog.js (its rightful home)
* Make json files autoformatted always
* Granular progress update with execution status text in in the starlark package run log view

## Is this change user facing?
YES

### CODE REVIEW

File: engine/frontend/src/component/log/CreateEnclaveLog.js [hunk 1]
```diff
@@ -32,12 +86,27 @@ export const CreateEnclaveLog = ({packageId, enclave, args, appData}) => {
...
     const readStreamData = (result) => {
+        // console.log(`${result.case} ${JSON.stringify(result.value)}`)
         if (result.case === INSTRUCTION) {
             setLogs(logs => [...logs, result.value.executableInstruction])
         }
 
+        if (result.case === PROGRESS_INFO && (result.value.currentStepInfo[0] === EXECUTION_IN_PROGRESS || result.value.currentStepInfo[0] === STARTING_EXECUTION)) {
+            const text = `${result.value.currentStepInfo[0]}: ${result.value.currentStepNumber} / ${result.value.totalSteps}`
+            // console.log(text)
+            let progress = ((result.value.currentStepNumber + 1) / (result.value.totalSteps + 1)) * 100
```
Comment (tedim52) [inline engine/frontend/src/component/log/CreateEnclaveLog.js]: i wonder why the `+1` is needed

Comment (tedim52) [reply engine/frontend/src/component/log/CreateEnclaveLog.js]: i assume to prevent division by zero?

**Review by tedim52 — APPROVED** [review]
LGTM

File: engine/frontend/src/component/log/CreateEnclaveLog.js (hunk 1 above)
Comment (adschwartz) [reply engine/frontend/src/component/log/CreateEnclaveLog.js]: I changed this logic

**Pull request merged**